src/components/CollectivitesExploitantsPage.style.tsx

```diff
@@ -15,6 +15,10 @@ export const FullWidthImage = styled.img<{ width?: string }>`
 export const CampagneAfficheImage = styled.img`
   margin-top: -120px;
   margin-bottom: -140px;
+  max-width: 100%;
+  @media (max-width: 992px) {
```

Comment:
je vois qu'on duplique les breakpoints un peu partout et je peux proposer 2 solutions pour cela

- la plus facile: Avec un fichier partagé de constantes comme ici https://dev.to/collegewap/how-to-use-media-queries-in-styled-components-54b7
- la mieux: en creant un theme qui ajoute cette fonctionnalité (de cette manière, pas besoin d'import supplémentaire)

Reply:
@martinratinaud super bonne idée, on se fait un ticket Trello ? Parce que ça va concerner pas mal d'endroits !